docroot/modules/custom/va_gov_backend/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -550,6 +551,20 @@ public function toggleFieldOfficeHours(array &$form, FormStateInterface $form_st
     }
   }
 
+  /**
+   * Remove delete button from edit forms.
+   */
+  public function removeDeleteButton(array &$form) {
+    // Users with any role, other than administrator role,
+    // shouldn't be able to delete anything.
+    if (!$this->userPermsService->hasAdminRole(TRUE)) {
+      // Remove the delete button.
+      if (isset($form['actions']['delete'])) {
+        $form['actions']['delete']['#access'] = FALSE;
+      }
```

Comment:
This should be just one if.   

```suggestion
    if (!$this->userPermsService->hasAdminRole(TRUE) && isset($form['actions']['delete']) {
      // Remove the delete button.
      $form['actions']['delete']['#access'] = FALSE;
```